### DISCUSSION TITLE
Expose more video transcoding options and add related documentation

### DISCUSSION THREAD

**Discussion by mgogoulos** [mediacms-io/mediacms#637] [Feature Requests]
**Describe the feature you'd like**
After discussing with Yaroslav on https://github.com/mediacms-io/mediacms/issues/232 I realize that 

- [ ] a few options related to video transcoding could be exposed on the admin interface, for example ffmpeg presets
- [ ] video transcoding related documentation needs be added (on the Administrator section of the docs)

**Comment by slhck** [comment]
You might want to have a look at https://github.com/svt/encore as either

- an example on what kind of options are useful to expose, or
- potentially simply reusing this as a backend in the long run

At the very least, I think it would be good to have the users be able to select:

- The resolutions that are being encoded
- Video and audio bitrates for different codecs/resolutions (i.e. defining the "bitrate ladder")
- Additional options applied to videos such as video filters
- The type of scaler used for the `scale` command
